merlin.psi.ch/cpu/computer-setup.yaml

```diff
@@ -0,0 +1,18 @@
+---
+label: merlin6
+hostname: merlin-l-01.psi.ch
+description: Merlin6 HPC at PSI
+transport: ssh
+scheduler: slurm
+work_dir: /home/{username}/aiida_run/
```

Comment:
Can you confirm that they don't provide `/scratch` folder?

Reply:
Thanks! There are two scratch folders:
- /scratch, for jobs running in one node.
- /shared-scratch, for jobs running on multiple nodes.
I change it to `/shared-scratch`.